3_calibrate/src/calibration_utils.R

```diff
@@ -10,14 +10,15 @@
 run_glm_cal <- function(nml_obj,
                         sim_dir,
                         cal_parscale = c('cd' = 0.0001, 'sw_factor' = 0.02, 'Kw' = 0.01),
-                        model_config){
+                        model_config,
+                        optimize = F){
```

Comment:
ticky tacky, but for consistency, preferred to spell out `FALSE` here in the future (probably not worth the change here unless you are making other edits)

Reply:
Done!